x-pack/plugins/triggers_actions_ui/public/application/sections/translations.ts

```diff
@@ -88,3 +88,48 @@ export const FEATURE_LABEL = i18n.translate(
     defaultMessage: 'Feature',
   }
 );
+
+const SHOW_REQUEST_MODAL_EDIT = i18n.translate(
+  'xpack.triggersActionsUI.sections.showRequestModal.subheadingTitleEdit',
+  {
+    defaultMessage: 'edit',
+  }
+);
+const SHOW_REQUEST_MODAL_CREATE = i18n.translate(
+  'xpack.triggersActionsUI.sections.showRequestModal.subheadingTitleCreate',
+  {
+    defaultMessage: 'create',
+  }
+);
+export const SHOW_REQUEST_MODAL_SUBTITLE = (edit: boolean) =>
+  i18n.translate('xpack.triggersActionsUI.sections.showRequestModal.subheadingTitle', {
+    defaultMessage: 'This elasticsearch request will {requestType} this rule.',
```

Comment:
```suggestion
    defaultMessage: 'This Kibana request will {requestType} this rule.',
```

Reply:
Resolved in this commit [1198485](https://github.com/elastic/kibana/pull/180085/commits/1198485fc2f8806aba05cfa7b1eaeadadd3c9076)